capstone/src/main/java/com/google/univiz/api/MapsResource.java

```diff
@@ -4,8 +4,8 @@
 import java.util.List;
 
 /**
- * The MapsResource interface is responsible for all data management/processing specific for the maps
- * feature of the UniViz webapp.
+ * The MapsResource interface is responsible for all data management/processing specific for the
+ * maps feature of the UniViz webapp.
```

Comment:
It looks like your only change to this file is javadoc reformatting. Considering that it has now moved to the `api.resource` package you can revert your reformatting change to avoid extra conflict resolution. Or just move the class in our local branch has well.